test-app/package.json

```diff
@@ -36,6 +36,7 @@
     "@embroider/test-setup": "^1.8.0",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
+    "@tsconfig/ember": "^2.0.0",
```

Comment:
looks like we should remove this from the package.json or from the ember-try scenario. I tend to remove it from the `package.json` in scope of the current `test-app`

Reply:
See comment above.